src/update-promoted-values.ts

```diff
@@ -83,6 +83,9 @@ export async function updatePromotedValues(
   for (const [serviceName, commits] of promotes.map((p) => p.relevantCommits)) {
     relevantCommits.set(serviceName, commits);
   }
+
+  logger.info(`Relevant commits: ${JSON.stringify(relevantCommits)}`);
```

Comment:
more logging for in progress debugging. will remove when everything is working great in prod